src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -27,4 +29,36 @@ public UserResponseDto findByEmail(@RequestParam String email) {
                 () -> new RuntimeException("User with email " + email + " not found"));
         return userResponseDtoMapper.mapToDto(user);
     }
+
+    @GetMapping("/")
+    public List<UserResponseDto> getAll() {
+        return userService.getAll().stream()
+                .map(userResponseDtoMapper::mapToDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/inject")
+    public String injectMockData() {
+        User bob = new User();
+        bob.setEmail("Bob@gm.com");
+        bob.setPassword("Anderson");
+
+        User alice = new User();
+        alice.setEmail("Alice@gm.com");
+        alice.setPassword("Waterspoon");
+
+        User david = new User();
+        david.setEmail("David@gm.com");
+        david.setPassword("Jonson");
+
+        User mike = new User();
+        mike.setEmail("Mike@gm.com");
+        mike.setPassword("Shoolermaster");
+
+        userService.add(bob);
+        userService.add(alice);
+        userService.add(david);
+        userService.add(mike);
```

Comment:
let's maybe register them instead?

Reply:
Ok. I create class InjectController for first enter in DB with Alice and Bob... But I can`t understand how to make the first login (base of User is empty - so I can`t enter in DB))

Reply:
https://www.toolsqa.com/postman/basic-authentication-in-postman/
check this article with screenshots